specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -3109,7 +3109,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_PerformMaintenance",
-        "description": "The operation to perform maintenance on a virtual machine.",
```

Comment:
Could you also update the description of the previous versions?

Reply:
Which all versions would you like me to update?

Reply:
All previous versions and new version (compute.json is now updated to 2019-12-01)